### DISCUSSION TITLE
Hoàn chỉnh bài báo cáo

### DISCUSSION THREAD

**Discussion by baonhu1304** [baonhu1304/Lop-10A2-nhom-1#14] [Show and tell]
Mụi ngừi dzô xem bài dzới bổ sung 1 phần nhỏ trong ó giúp tuôi nhe.
Đánh máy dzô đây nè để tui chỉnh dzô file word cho dễ.
Cảm ưn mấy bà trc nghen

**Comment by thaopham2501** [comment]
qua máy t nó bị lệch tùm lum
